python/mosaic/api/functions.py

```diff
@@ -43,6 +43,8 @@
     "st_ymax",
     "st_zmin",
     "st_zmax",
+    "st_x",
+    "st_y",
 
```

Comment:
@milos-colic so there is no `st_z` but there is now `st_zmin` and `st_zmax`?

Reply:
yes that is correct, st_zmin and st_zmax will be deprecated in a separate PR.
We can introduce it back, but for 3D geoms when we have a proper way of handling them.